app/controllers/searches_controller.rb

```diff
@@ -12,7 +12,7 @@ def new
   end
 
   def create
-    @search = Search.new(filter:, term:, dob:, version2: version_2?)
+    @search = Search.new(filter:, term:, dob:, version2: true)
```

Comment:
at the end of removing all feature flags,
`version2: true` arg can be removed.

Reply:
Ah, it's because the `hearing` flag also sometimes is used to decide whether to use `version2` in Search?

Reply:
Yes,
`DEFENDANTS_SEARCH_CDAPI_V2` might have been a better name than `DEFENDANTS_SEARCH.`